.mergify.yml

```diff
@@ -1,46 +1,34 @@
 pull_request_rules:
-  - name: 'rebase unreviewed non-release PRs'
+  - name: "rebase unreviewed non-release PRs"
     conditions:
       - head~=^(?!(release|hotfix)).*$
-      - status-success=Travis CI - Branch
-      - status-success=Travis CI - Pull Request
-      - status-success=SonarCloud Code Analysis
-      - status-success=verification/cla-signed
-      - "#approved-reviews-by<=0"
-      - "#changes-requested-reviews-by<=0"
+      - "#approved-reviews-by=0"
```

Comment:
Could you point me to the docs where it says this is only needed for sets? Because according the [attributes section](https://doc.mergify.io/conditions.html#attributes) `approved-reviews-by` is an array of strings, and according to the [operators section](https://doc.mergify.io/conditions.html#operators) `=` checks for strict equality among the array values.

Reply:
https://doc.mergify.io/conditions.html#operators

> This operator checks for the value to be lesser then or equal to the provided value. It’s usually used to compare against the length of an array using the # prefix.

Hmm, maybe I misinterpreted this wrongly this monday morning...

But regardless, the examples always check `#{$array}` with `=`
https://doc.mergify.io/examples.html#require-all-requested-reviews-to-be-approved